<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>新密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pass">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sure">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>密保问题：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>your birthday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>your fa birth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>your ma birth</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>密保答案：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-20</y>
      <width>491</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/59f7e4619b93e_1024.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_ID</zorder>
   <zorder>lineEdit_Pass</zorder>
   <zorder>pushButton_Sure</zorder>
   <zorder>label</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
